play/PlayerRules/Basic Rules.txt

```diff
@@ -194,8 +194,8 @@ and the score of your roll would be
 That's a really high roll! But that may or may not be enough to lift the object
 depending on how heavy it is!
 
-Ranged or melee weapon accuracy bonuses use slightly different equations, as combat
-accuracy rolls are played out on a d10 scale:
+Ranged and melee weapon accuracy bonuses use a slightly different equation than the 
+above Stat Bonus equation, as combat accuracy rolls are played out on a d10 scale:
 

```

Comment:
I guess @trowl223 you could be referring to a couple of changes, I put the equation in because of the d10 accuracy change (like Michael referenced). The equation is different in more than just scale because I wanted to make base stats less powerful in comparison to attachments when you min-max.